### DISCUSSION TITLE
Password Resets

### DISCUSSION THREAD

**Discussion by Shawn-E** [supabase/supabase#11576] [Questions]
I am trying to reset the users password but evertime I call the update user function it says `AuthSessionMissingError: Auth session missing!`. Here is the password reset code
```js
app.post('/auth/reset-password', async function(req, res){
    const access_token = req.cookies.token
    const refresh_token = req.cookies.rtoken
    const { data2, error2 } = supabase.auth.setSession({
        access_token,
        refresh_token
    });
    const { data, error } = await supabase.auth.updateUser({
        password: req.body.password
    });
    console.log(data, error)
})
```
How could I got about fixing this?

**Comment by GaryAustin1** [comment]
If this is a server side function make sure you have persistSession false when you initialize the client.

https://github.com/supabase/gotrue-js/issues/539
